# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../util-linux/no_tqueue.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./sys-utils/cytune.c.orig	Mon Apr 22 18:51:56 2002
+++ ./sys-utils/cytune.c	Mon Apr 22 18:52:14 2002
@@ -54,7 +54,7 @@
 #include <termios.h>
 
 #include "../defines.h"		/* for NEED_tqueue_h */
-#ifdef NEED_tqueue_h
+#ifdef NEED_tqueue_h_disabled
 #include <linux/tqueue.h>	/* required for old kernels (for struct tq_struct) */
 				/* compilation errors on other kernels */
 #endif
